backend/cors-json-file.json

```diff
@@ -0,0 +1,6 @@
+[{
```

Comment:
Where do you use it?

Reply:
It's required when you setup CORS on gcloud using the command line tools.